src/ampl.js

```diff
@@ -1,33 +1,29 @@
-var Remarkable = require('remarkable');
-var htmlparser = require("htmlparser2");
+import Remarkable from 'remarkable';
+import htmlparser from 'htmlparser2';
 
-var remarkable = new Remarkable('full');
+import { getDims, updateImgTags } from './imageDims.js';
+import { createAmpPage } from './templates.js';
 
-import { getDims } from './imageDims.js';
-import { html2Amp } from './templates.js';
+const remarkable = new Remarkable('full');
 
-export function parse(mdString, css, callback) {
-  var htmlRaw = remarkable.render(mdString);
-  var htmlAmp = html2Amp(css, htmlRaw)
-  parseHtml(htmlAmp, function(htmlAmp, data) {
-    getDims(data.urls, function(dimensions) {
-      // todo remove while loop
-      var i = 0;
-      var imageTagRegex = /(<img)/;
-      while(imageTagRegex.test(htmlAmp)) {
-        var newTag = `
-          <amp-img
-            width="${dimensions[i].width}"
-            height="${dimensions[i].height}"
-            layout="responsive"`;
-        htmlAmp = htmlAmp.replace(imageTagRegex, newTag);
-        i += 1;
-      }
-      callback(htmlAmp);
-    });
-  });
+export const parse = (markdown, optsOrStyle, callback) => {
+  const opts = typeof optsOrStyle === 'string' ?
+    {style: optsOrStyle} : optsOrStyle;
+  markdown2AmpHTML({ markdown }, HTML =>
```

Comment:
I put this in an object so that if I want to add further arguments to the function `markdown2AmpHTML` it can be backward compatible (since this function will be exposed to the module's users)